src/lib/pubkey/sphincsplus/sphincsplus_common/sp_wots.cpp

```diff
@@ -0,0 +1,191 @@
+/*
+* WOTS+ - Winternitz One Time Signature+
+* (C) 2023 Jack Lloyd
+*     2023 Fabian Albert, René Meusel, Amos Treiber - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+**/
+
+#include <botan/internal/sp_hash.h>
+#include <botan/internal/sp_wots.h>
```

Comment:
`sp_wots.h` should be the first include with a newline separating it from the rest of the includes